src/vendor/ramda.ts

```diff
@@ -0,0 +1,12 @@
+/**
+ *  Explicit list of helper functions in order to reduce bundle size
+ */
+
+export { default as sortBy } from 'ramda/src/sortBy';
```

Comment:
Why is this necessary? Can't we just use regular tree-shaking?

Reply:
It seems like it doens't matter wether we `import * as R from 'ramda'` or `import { path } from 'ramda'. Both escape Tree shaking. https://stackoverflow.com/a/45746950/2057532
This is why we started importing from the `ramda/src`.
To avoid the multiple lines, I find it simpler to place all the imports in `vendor`. It would also reduce the amount of import changes whenever they decide to support Tree shaking.